modules/java-modules/elastic-v2/src/main/java/org/syslog_ng/elasticsearch_v2/client/http/ESHttpsClient.java

```diff
@@ -112,7 +116,7 @@ private SSLContextBuilder setupSSLContextBuilder(ElasticSearchOptions options) {
         SSLContextBuilder sslContextBuilder = new SSLContextBuilder();
         KeyStore trustStore = null;
 
-        if (options.getJavaSSLInsecure()) {
+        if (!isSSLInsecure(options)) {
```

Comment:
this does the opposite you wanted, remove the negation"!":
"if (isSSLInsecure(options))"